Leetcode/intervals.md

```diff
@@ -0,0 +1,27 @@
+# Intervals
```

Comment:
Your Pull Request title: LEETCODE Added non-overlapping intervals solution

Check result:

* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']